### PR TITLE
Use adminimal as default theme.

### PR DESCRIPTION
**Pull request by kasperg**
Sooo pretty.

### CODE REVIEW

File: web/config/sync/core.extension.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ module:
   dynamic_page_cache: 0
   field: 0
   filter: 0
+  help: 0
```
Comment (kallehauge) [inline web/config/sync/core.extension.yml:9]: This is a bit out of the commits scope (enable module)

**Comment by kallehauge** [comment]
When the "help" module is fixed (just a separate commit, not necessarily a new PR), then I'm satisfied and you can merge :relaxed:

File: web/config/sync/core.extension.yml (hunk 1 above)
Comment (kasperg) [inline web/config/sync/core.extension.yml:9]: [Adminimal depends on the help module](https://www.drupal.org/node/2629744) so enabling the help module is a requirement. I have updated the commit comment to explain this.

**Comment by kallehauge** [comment]
:+1: I approve of this change

**Pull request merged**